docs/faq/index.md

```diff
@@ -0,0 +1,134 @@
+# FAQ
+
+## Installation
+
+### Setup fails
+
+- In case you encounter errors during setup or build process, please make sure to use an appropriate branch for your environment
+  - ROS 2 Galactic, Ubuntu 20.04: `galactic` branch
+  - ROS 2 Humble, Ubuntu 22.04: main branch
+- It's also important to delete `./build` `./install` and `./src` directory before rebuilding CARET if you have built CARET using wrong settings
+
+### CLI tool doesn't work
+
+In case CLI tool execution fails, please make sure to perform CARET environment settings
+
+```bash
+source /opt/ros/humble/setup.bash
+source ~/ros2_caret_ws/install/local_setup.bash
+
+ros2 caret check_caret_rclcpp --workspace <path-to-workspace>
```

Comment:
@takeshi-iwanari 
Could you add caution for  Anaconda-based Python?
CLI tools doesn't work property in anaconda environment.
https://github.com/tier4/caret/issues/37#issuecomment-1276952560

Reply:
https://github.com/tier4/CARET_doc/pull/89/commits/e9b5d479466fed28efee8c311c25d300fea67490 adds the caution.